web_widget_bokeh_chart/README.rst

```diff
@@ -50,8 +50,6 @@ To insert a Bokeh chart in a view proceed as follows:
             p = figure()  # import that as `from bokeh.plotting import figure`
             line = p.line([0, 2], [1, 8], line_width=5)
             # (...)
-            # `p.html.data` contains both markup and the script of a chart.
-            rec.bokeh_chart = p.html.data
```

Comment:
You need assign something to the bokeh_chart field (you are inside a computed field). Am I missing something?

This would work in the latest version:
```
script, div = components(p)
rec.bokeh_chart = '%s%s' % (div, script)
```

Reply:
thanks